dns.go

```diff
@@ -206,6 +206,22 @@ func (c *Client) GetDNSRecord(domainID, name string) (*DNSRecord, error) {
 	return nil, ErrDNSRecordNotFound
 }
 
+// UpdateDNSRecord updates the DNS record
+func (c *Client) UpdateDNSRecord(rc *DNSRecordConfig, d *DNSDomain, r *DNSRecord) (*DNSRecord, error) {
+	url := fmt.Sprintf("/v2/dns/%s/records/%s", d.ID, r.ID)
+	body, err := c.SendPutRequest(url, rc)
+	if err != nil {
+		return nil, err
+	}
+
+	var dnsDomain = &DNSRecord{}
```

Comment:
This variable should be called `dnsRecord` rather than `dnsDomain` (as it's a Record object not a Domain)